docs/glossary/glossary.md

```diff
@@ -9,11 +9,11 @@ Here's a list of the most commonly used terms:
 | ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----- |
-| Kyma Cluster        | A Kubernetes cluster                                                                                                                                                                                                                                                                                                                                                                                                                                                                |       |
-| Function            | A simple code snippet that you can run without provisioning or managing servers. It implements the exact business logic you define. A Function is based on the Function custom resource and can be written in either Node.js or Python. A Function can perform a business logic of its own. You can also bind it to an instance of a service and configure it to be triggered whenever it receives a particular event type from the service or a call is made to the service's API. |       |
-| Microservice        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |       |
+| Kyma Cluster        | A Kubernetes cluster with Kyma installed. A cluster is a set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.                                                                                                                                                                                                                                                                                                                                                                                                                                                                | [Kubernetes Components](https://kubernetes.io/docs/concepts/overview/components/) |
```

Comment:
Or, alternatively, instead of repeating the cluster definition here, we could also just leave the first sentences and link to the Kubernetes definition of `cluster`, like this: 
```suggestion
| Kyma Cluster        | A [Kubernetes cluster](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cluster) with Kyma installed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | [Kubernetes Components](https://kubernetes.io/docs/concepts/overview/components/) |
```